src/components/LineChart.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { Line } from "react-chartjs-2";
+import { Chart as ChartJS, registerables } from "chart.js";
+ChartJS.register(...registerables);
+
+function LineChart({ weathers, active, toggle }) {
+  const [showTemp, setShowTemp] = useState([]);
+  useEffect(() => {
+    const fullTemp =
+      weathers &&
+      weathers.map((data) => {
+        if (toggle) {
+          return Math.round(data.main.temp * 1.8 + 32);
+        } else {
+          return Math.round(data.main.temp);
+        }
```

Comment:
Có thể viết như này cho gọn hơn nè
```suggestion
      weathers && weathers.map(data => toggle ? Math.round(data.main.temp * 1.8 + 32) : Math.round(data.main.temp))
```